### PR TITLE
5 -  Draw a database model for each microservice

### PR DESCRIPTION
**Pull request by bojanabogicevic**
Add database models for product, order and user services

### CODE REVIEW

**Review by ilicstef94 — CHANGES_REQUESTED** [review]

File: order-service/documentation/database/order-service-model.drawio
[the remark this answers is not in the document]
Comment (bojanabogicevic) [reply order-service/documentation/database/order-service-model.drawio:1]: Done

File: product-service/documentation/database/product-service-model.drawio
[the remark this answers is not in the document]
Comment (bojanabogicevic) [reply product-service/documentation/database/product-service-model.drawio:1]: First part is done, regarding Review class, it is currently not defined as entity therefore it shouldn't be included in the first database model.

File: user-service/documentation/database/user-service-model.drawio
[the remark this answers is not in the document]
Comment (bojanabogicevic) [reply user-service/documentation/database/user-service-model.drawio:1]: Done

File: product-service/documentation/database/product-service-model.drawio
[the remark this answers is not in the document]
Comment (ilicstef94) [reply product-service/documentation/database/product-service-model.drawio:1]: I saw that there are some classes that are also not defined as entities but are in the entities directory. I created a ticket to define all entity classes uniformly

**Review by ilicstef94 — APPROVED** [review]

**Review by bogdanmilojevic — APPROVED** [review]

**Pull request merged**